vba_timeseries/models/nonu/suuvid_obs_nonu.m

```diff
@@ -23,7 +24,7 @@
 Qtot = sum(Qcur); %total value
 
 %probability of responding at all
-p_respond = phi_tb/(1 + exp(-gamma * (Qtot)));
+p_respond = phi_tb/(1 + exp(-gamma * (Qtot - nu)));
```

Comment:
Same story here -- the nonu observation function should not add nu. This should be in the value2pl folder.